kubernetes/client/rest.py

```diff
@@ -41,11 +41,11 @@ def __init__(self, resp):
 
     def getheaders(self):
         """Returns a dictionary of the response headers."""
-        return self.urllib3_response.getheaders()
```

Comment:
this file is generated by openapi-generator. Could you try sending a fix to the upstream? Otherwise we need to carry a patch to make sure this change doesn't get reverted during new releases

Reply:
Hmmm, after doing some more investigation, looks like upstream urllib3's v2.0 release is significantly delayed (was supposed to be released in 2021). I think at least for the immediate future, it would be better for us to just silence the specific deprecation warning on our (ACK's) side. I'll close out the PR and issue.